kbase-extension/static/kbase/js/common/jobs.js

```diff
@@ -79,11 +178,36 @@ define([
                 cssClass = '';
         }
 
-        return span({
-            class: `${cssBaseClass}__summary${cssClass}`,
-        }, label);
+        return span(
+            {
+                class: `${cssBaseClass}__summary${cssClass}`,
+            },
+            label
+        );
     }
 
+    /**
+     * Ensures that the child_jobs data is in the correct place in the jobState object
+     *
+     * It should be possible to remove this function after migrating to ee 2.5
+     * @param {object} jobState
+     * @return {object} jobState, possibly edited
+     */
+    function updateJobModel(jobState) {
```

Comment:
temporary hack -- once the backend is updated to use the new format for jobs data, this function can disappear